ieeg_filtered_speech/sub-cm4/ses-01/sub-cm4_ses-01_ieeg.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Many missing or incorrectly labeled fields, see spec. Why 64 eeg channels? no iEEGsurf? no iEEGdepth? why MEG/MEGREF, and several others
Add manufacturer

Reply:
yep - these are changes I need to add to MNE-BIDS, work on that in the PR here:

https://github.com/mne-tools/mne-bids/pull/28